src/shared/varlink-io.systemd.Machine.c

```diff
@@ -4,6 +4,13 @@
 
 #include "varlink-io.systemd.Machine.h"
 
+#define SD_VARLINK_LOOKUP_AND_POLKIT_INPUT_FIELDS                                                                                              \
```

Comment:
the SD_ prefix is only used for our public interfaces, those that other projects are supposed to consume. This here is an internal macro, hence drop the SD_ prefix here